.github/workflows/CI.yaml

```diff
@@ -11,7 +11,6 @@ on:
 
 env:
   CARGO_INCREMENTAL: 0
-  CARGO_REGISTRIES_CRATES_IO_PROTOCOL: sparse
```

Comment:
I'd update toolchains to the latest version instead of moving backward.

Reply:
the mac-os job installs somehow stable version of 1.67.1. Seems it downloads under the hood the rustup installer of that version. Dunno why not later version. I'd suggest removing until really needed the sparse/the later stable version is downloaded by default.

Reply:
`actions-rs/toolchain` updates toolchain to the latest version. But up to you.

Reply:
discussed.